pkg/kubelet/container/runtime.go

```diff
@@ -57,7 +57,10 @@ type Runtime interface {
 	Version() (Version, error)
 	// APIVersion returns the API version information of the container
 	// runtime. This may be different from the runtime engine's version.
+	// TODO(random-liu): We should fold this into Version()
 	APIVersion() (Version, error)
+	// CheckStatus checks whether the underlying runtime is working normally.
+	CheckStatus() error
```

Comment:
If it is called `Status()`, I feel like it will return some kind of "status", but it only returns error or not. While, if it is called `CheckStatus()`, I could understand that it is trying to check the status, and may encounter some error during checking.
I'm not good at naming, just MHO. :)